Sources/AWSLambdaRuntimeCore/ControlPlaneRequest.swift

```diff
@@ -54,8 +54,7 @@ struct Invocation: Hashable {
         self.requestID = requestID
         self.deadlineInMillisSinceEpoch = unixTimeInMilliseconds
         self.invokedFunctionARN = invokedFunctionARN
-        self.traceID = headers.first(name: AmazonHeaders.traceID) ?? 
-            "Root=\(AmazonHeaders.generateXRayTraceID());Sampled=0"
+        self.traceID = headers.first(name: AmazonHeaders.traceID) ?? "Root=\(AmazonHeaders.generateXRayTraceID());Sampled=0"
```

Comment:
should we add a small unit test too so this does not regress?

Reply:
testing that we generate a default traceID if no header is provided

Reply:
I have that ready. I used the header from another file and changed the year to 2022, but now soundness.sh is complaining that it's not an acceptable year. Should I add `s/2022/YEARS/` to the `replace_acceptable_years()` function?

Reply:
See 374dc71